src/CodeConventions.cs

```diff
@@ -0,0 +1,59 @@
+namespace ConsoleApp;
+
+interface IInterface
+{
+    public void Action();
+    public void Show();
+}
+
+public class ClassName
+{
+    public static int PublicStaticIntProperty { get; set; }
+    private static int PrivateStaticIntProperty { get; set; }
+    internal static int InternalStaticIntProperty { get; set; }
+    protected static int ProtectedStaticIntProperty { get; set; }
+
+
+    public static int PublicStaticIntField;
+    private static int _privateStaticIntField;
+    internal static int _internalStaticIntField;
+    protected static int _protectedStaticIntField;
+
+    public int PublicIntProperty { get; set; }
+    private int PrivateIntProperty { get; set; }
+    internal int InternalIntProperty { get; set; }
+    protected int ProtectedIntProperty { get; set; }
+
+    public int PublicIntField;
+    private int _privateIntField;
+    internal int _internalField;
+    protected int _protectedIntField;
+
+
+    public ClassName(int FirstArgument, int SecondArgument, int ThirdArgument, int ForthArgument, int FifthArgument)
```

Comment:
code convention violated
```suggestion
    public ClassName(int firstArgument, int secondArgument, int thirdArgument, int forthArgument, int fifthArgument)
```